examples/history-api-fallback/file

```diff
@@ -0,0 +1 @@
+file
```

Comment:
May be better to add
```
devServer: {
		historyApiFallback: {
			disableDotRule: true
		}
	}
```
 to `webpack.config.js`? In this case it will be shown, that rewrite to index.html occurs before trying to serve file.txt.